libflux/flux/benches/everything.flux

```diff
@@ -1047,6 +1047,16 @@ builtin table : (<-tables: [A], columns: C) => [B] where A: Record, B: Record, C
 //         The length of the interval. This defaults to the
 //         every duration.
 builtin window : (<-tables: [A], ?time: string, every: duration, ?period: duration, columns: C) => [B] where A: Record, B: Record, C: Record
+builtin implicitWindow : (
```

Comment:
I don't know if we need to add this. We aren't really testing this interface at all. Maybe one day we'll just re-generate `everything.flux` in which case this will get included, but I don't think we should be manually adding things to `everything.flux`.